### PR TITLE
Waveform Refresh Mode

### PR DESCRIPTION
**Pull request by skks1212**
Added refresh mode to Waveform

### CODE REVIEW

**Comment by nihal467** [comment]
@skks1212 
image

https://deploy-preview-3768--care-net.netlify.app/facility/c4911f55-d4f7-4b36-bbcb-9b115d1bfe2c/patient/88986c19-3ab5-4820-b35d-7b7d95ac4f00/consultation/4f80a4cb-ed21-4b8f-95fd-b9c3a74e851e

the wave is keep on breaking.

**Comment by gigincg** [comment]
@nihal467 This is expected behaviour.

**Comment by nihal467** [comment]
@skks1212 @gigincg 

https://screen-recorder-bucket.s3.ap-south-1.amazonaws.com/ScreenRecorder_2022-11-09_c8bf154b-9a7c-4ec1-9acc-06cbb966748a.mp4

- [ ] there is a shivering in the vital, which doesn't look great at all

 **the waveform is good is normal view and responsive view. After todays EOD, will finalise the approval status of the PR**

**Comment by nihal467** [comment]
conditionally approving it, the existing issue will be made a new pr

File: src/Components/Facility/Consultations/components/LinePlot.tsx [hunk 1]
```diff
@@ -102,19 +102,27 @@ export const LinePlot = (props: any) => {
         text: "",
       },
       grid: {
-        left: "15px",
-        right: "30px",
+        show: false,
+        borderColor: "transparent",
+        left: "40px",
+        right: "20px",
```
Comment (khavinshankar) [inline src/Components/Facility/Consultations/components/LinePlot.tsx:109]: @skks1212 changing LinePlot is causing side effects in summary plots
image

**Review by khavinshankar — CHANGES_REQUESTED** [review]

File: src/Components/Facility/Consultations/components/LinePlot.tsx (hunk 1 above)
Comment (shivankacker) [reply src/Components/Facility/Consultations/components/LinePlot.tsx:109]: Looks like a result of an earlier PR. Il fix it asap.

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**